Makefile

```diff
@@ -0,0 +1,42 @@
+# This Makefile is meant to be used by people that do not usually work
+# with Go source code. If you know what GOPATH is then you probably
+# don't need to bother with make.
+
+GOBIN = $(shell pwd)/build/bin
+GO ?= latest
+
+
+swarm:
+	build/env.sh go run build/ci.go install ./cmd/swarm
+	@echo "Done building."
+	@echo "Run \"$(GOBIN)/swarm\" to launch swarm."
+
+all:
+	build/env.sh go run build/ci.go install
+
+test: all
+	build/env.sh go run build/ci.go test
+
+lint: ## Run linters.
+	build/env.sh go run build/ci.go lint
+
+clean:
```

Comment:
I'd rather we don't maintain any of these to be honest. Whoever works on this project, should be aware of the caches within Go. Also this is about Go 1.10 which is no longer supported by the Go team - we suggest people use Go 1.11 or Go 1.12.